packages/@atjson/offset-annotations/src/annotations/sidebar.ts

```diff
@@ -0,0 +1,14 @@
+import { BlockAnnotation } from "@atjson/document";
+
+/**
+ * Sidebar annotations are alignable and may be used to display
+ * content within a body that is graphically separate but with
+ * contextual connection.
+ **/
+
+export class Sidebar extends BlockAnnotation<{
+  inset?: "left" | "right";
```

Comment:
Perhaps we should just make this required instead of trying to set a default value.